### PR TITLE
Multitenancy UI Polish

### PR DESCRIPTION
**Pull request by alexpaxton**
- [x] Rebased/mergable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

Connect #2417 
Connect #2436 
Connect #2437 

### The Problem
- When setting up multiple orgs users as a `superadmin` it feels tedious to switch orgs 
- `ID` Column in Organizations table is unecessary
- `superadmin` indicator in user nav item is unecessary
- Provider and Scheme is not displayed in the UI when logged in
- Hard to locate self in chronograf admin users table

### The Solution
- Add org switching UI to organizations table
- Remove `ID` column from organizations table
- Remove `You are a SuperAdmin` indicator from user nav item
- Add `[scheme] / [provider]` to user nav item
- Highlight self in chronograf admin users table

### Preview
screen shot 2017-12-04 at 1 19 23 pm
Org switching UI: borrowed the `shuffle` icon from Grafana. No more `ID` columns.

screen shot 2017-12-01 at 3 58 55 pm
Listing `scheme` & `provider` instead of super admin indicator

screen shot 2017-12-01 at 5 16 28 pm
Highlight self

### CODE REVIEW

**Comment by alexpaxton** [comment]
Came across #2463 during the work on this branch, decided to file as a separate issue since the work seems signficant

**Comment by jaredscheib** [comment]
@alexpaxton In addition to simply needing to know how I got into the app, another use case for knowing which scheme & provider I'm authenticated through is that i want to know which user I am in the list when modifying users. Given that this has been added to the side nav, this means I'll still have to have the side nav open to know this info while modifying users.

I think it's good for that info to be in the side nav (and personally I think superadmin is useful there), but it may also be good to display my username, provider, and scheme in the main app when I'm on the Users Table, or otherwise to indicate which user _I_ am in the users table clearly inline in the table itself, such as a different style for the user that is _I_.

And where does the indication of SuperAdmin now show up?

**Comment by jaredscheib** [comment]
According to @nhaugo and @gunnaraasen, you don't need to know if you're a superadmin. You should just have the powers that come with that territory. So I guess we don't need the indicator anywhere.

So 1 and 3 above remain, forget about 2.

**Comment by alexpaxton** [comment]
RIP superadmin indicator

**Comment by alexpaxton** [comment]
@jaredscheib addressed 1 in the bottom screenshot

3. Gonna keep messing with this part. I think icons alone aren't clear enough, and adding the column header helps slightly, but I have a feeling there's a better design out there

**Comment by jaredscheib** [comment]
I think `Switch to` or `Switch into` is more straightforward than `Switch` for the inline org switching. Otherwise it could seem that I'm performing a `switch` action on an organization, and I don't know what that means.

**Comment by gunnaraasen** [comment]
Just to be clear, I think there is a need for a user profile page of some kind where you can view basic info about the currently signed-in user, e.g. email, role, reset account password, etc. A user's role should be discoverable, but it's not super useful so shouldn't clutter up the main UI.

**Comment by jaredscheib** [comment]
Can you open an issue for that @gunnaraasen ?

**Comment by alexpaxton** [comment]
@jaredscheib copy updated

File: ui/src/admin/components/chronograf/OrganizationsTableRow.js [hunk 1]
```diff
@@ -119,8 +131,17 @@ class OrganizationsTableRow extends Component {
...
-        <div className="orgs-table--id">
-          {organization.id}
+        <div className="orgs-table--active">
+          {organization.id === currentOrganization.id
+            ? <button className="btn btn-sm btn-success">
+                <span className="icon checkmark" /> Current
+              </button>
+            : <button
+                className="btn btn-sm btn-default"
+                onClick={this.handleChangeCurrentOrganization()}
```
Comment (121watts) [inline ui/src/admin/components/chronograf/OrganizationsTableRow.js]: If you don't evaluate this function (remove the `()`) you can remove the first anonymous function in `handleChangeCurrentOrganization`.  So, line 41 would look like `handleChangeCurrentOrganization = async () => {`

Comment (alexpaxton) [reply ui/src/admin/components/chronograf/OrganizationsTableRow.js]: Ahh good catch, changed

**Comment by alexpaxton** [comment]
@121watts @jaredscheib made the changes

**Comment by jaredscheib** [comment]
Looks good but not sure if it resolves #2436. How do you feel about this hypothetical look:
screen shot 2017-12-04 at 3 11 19 pm

**Pull request merged**